### PR TITLE
Update toc.yml

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra  see [this comment](https://github.com/dotnet/AspNetCore.Docs/pull/27527#issuecomment-1306334745)
once @rafikiassumani-msft approves this PR, can you merge and then merge to live?

File: aspnetcore/toc.yml [hunk 1]
```diff
@@ -688,8 +688,6 @@ items:
         uid: web-api/advanced/conventions
       - name: Handle errors
         uid: web-api/handle-errors
-      - name: Port tunneling in VS to debug web APIs
-        href: /connectors/custom-connectors/port-tunneling
```
Comment (Rick-Anderson) [inline aspnetcore/toc.yml:692]: This is the API with controllers section. Should I revert this?
image

Comment (rafikiassumani-msft) [reply aspnetcore/toc.yml:692]: Do we need a TOC on API testing and including both HttpRepl, Port tunneling?

**Comment by tdykstra** [comment]
This PR as it stands deletes both occurrences of the Tunneling doc, including the one in the non-minimal section -- that's not what we want to do, is it?

**Comment by Rick-Anderson** [comment]
@tdykstra 
image

I need to add HttpRepl

**Comment by Rick-Anderson** [comment]
MOved to #27549

**Pull request closed without merging**